src/objects/overview/overview.scss

```diff
@@ -0,0 +1,56 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use '../../mixins/ms';
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/fluid';
+
+/**
+ * The Overview stacks content vertically on small screens.
+ * On large screens the header and actions are stacked horizontally above the content.
+ * The specific spacing values were inferred from the Article Listing prototype.
+ */
+
+.o-overview__header {
+  margin-bottom: ms.step(1);
+}
+
+.o-overview__actions {
+  margin-bottom: ms.step(3);
+}
+
+@supports (display: grid) {
+  @media (min-width: breakpoint.$l) {
+    .o-overview {
+      @include fluid.column-gap(
+        breakpoint.$s,
+        breakpoint.$xl,
+        size.$spacing-gap-fluid-min,
+        size.$spacing-gap-fluid-max
+      );
+
+      align-items: center;
+      display: grid;
+      grid-template-areas:
+        'header header actions'
+        'content content content';
+      grid-template-columns: repeat(3, minmax(max-content, 1fr));
```

Comment:
Why `max-content`?

Reply:
Good question. I'd copied that over from the prototype without thinking about it. This can be simplified.

Reply:
I simplified this to `grid-template-columns: repeat(3, 1fr);`. Let me know if you think a different value would make more sense.